src/rust/ensogl/example/src/list_view.rs

```diff
@@ -44,8 +44,8 @@ mod icon {
     use super::*;
     ensogl_core::define_shape_system! {
         (style:Style,id:f32) {
-            let width  = list_view::entry::ICON_SIZE.px();
-            let height = list_view::entry::ICON_SIZE.px();
+            let width  = 14.0.px();
+            let height = 14.0.px();
```

Comment:
I promised to left comment here, but I don't know any longer how you want to modify it, so here is just a ping :)